client/me/help/help-contact/index.jsx

```diff
@@ -420,6 +427,11 @@ const HelpContact = React.createClass( {
 		return isHappychatAvailable && olark.isUserEligible;
 	},
 
+	shouldUseDirectly: function() {
+		const isEn = i18n.getLocaleSlug() === 'en';
```

Comment:
Ah thanks! I missed that, was following along with [what I saw from `shouldUseHappychat()`](https://github.com/Automattic/wp-calypso/blob/c01e34173bfde6cef399ed8f1281fc0c535beb79/client/me/help/help-contact/index.jsx#L414) but I'll switch to the prop from Redux. :+1: